test/unit/vl-map-modify-action.test.html

```diff
@@ -0,0 +1,61 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <script src="../../../@webcomponents/webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../../web-component-tester/browser.js"></script>
+
+  <script type="module" src="../../src/vl-map-all.js"></script>
+  <script src="/node_modules/sinon/pkg/sinon.js"></script>
+</head>
+
+<body>
+<test-fixture id="vl-map-modify-action-fixture">
+  <template>
+    <vl-map>
+      <vl-map-features-layer>
+        <vl-map-modify-action>
+        </vl-map-modify-action>
+      </vl-map-features-layer>
+    </vl-map>
+  </template>
+</test-fixture>
+
+<script type="module">
+  import {VlMapModifyAction} from '../../src/vl-map-modify-action.js';
+  import {awaitUntil} from '/node_modules/vl-ui-core/dist/vl-core.js';
+
+  suite('vl-map-modify-action', () => {
+    const sandbox = sinon.createSandbox();
+
+    setup((done) => {
+      customElements.whenDefined('vl-map-modify-action').then(() => done());
+    });
+
+    teardown(() => {
+      sandbox.restore();
+    });
+
+    test('een modify actie is een map actie', () => {
+      assert.isTrue(VlMapModifyAction.isVlMapAction());
+    });
+
+    test('nadat de modify actie voltooid is, zal de onModify worden opgeroepen', async () => {
+      const map = fixture('vl-map-modify-action-fixture');
+      await map.ready;
+      const modifyAction = map.querySelector('vl-map-modify-action');
+      await awaitUntil(() => modifyAction.action != null);
+      let featureModified = false;
+      modifyAction.onModify(() => {
+        featureModified = true;
+      });
+      modifyAction.action.modifyInteraction.dispatchEvent({type: 'modifyend', features: ['feature']});
+
+      await awaitUntil(() => featureModified === true);
```

Comment:
Ik ben minder fan van duplicatie op extend niveau. Ik zou dan eerder één test toevoegen op het abstract niveau en via de fixture de extends testen. Op die manier test je evenveel maar is er geen test duplicatie.

Reply:
Bedoel je dan niet eerder bij de draw-line action ipv hier bij de onModify of wat bedoel je juist?

Reply:
Ik heb het aangepast, zie vl-map-draw-action.test.html.